2-mandatory/5-writing-tests.js

```diff
@@ -35,41 +35,57 @@ function convertScoreToGrade(score) {
   passes.
 */
 test("a score of 83 is grade A", () => {
-  expect(convertScoreToGrade(83), "Z");
+  expect(convertScoreToGrade(83), "A");
 });
 
 /*
   The rest of the tests have comments describing what to test and you need to
   write a matching test
 */
 
-test.skip("a score of 71 is grade B", () => {
+test("a score of 71 is grade B", () => {
+  expect(convertScoreToGrade(71), ("B"))
   /* Remove the .skip above, then write the test body. */
 });
 /*
   Write a test that checks a score of 68 is grade C
 */
```

Comment:
These are almost correct - but maybe double-check the names you've given to these tests - a few small mistakes.